server.go

```diff
@@ -1702,6 +1705,9 @@ func (s *Server) serveConn(c net.Conn) error {
 		if serverName != nil {
 			ctx.Response.Header.SetServerBytes(serverName)
 		}
+		if contentType != nil {
+			ctx.Response.Header.SetContentTypeBytes(contentType)
```

Comment:
But now it will always overwrite the user set content type with the default content type?